bridge/dd_init.php

```diff
@@ -7,6 +7,8 @@
 
 require_once __DIR__ . '/functions.php';
 
+// trigger configuration reload to memoize values of all configuration options as set by environment variables
+function_exists('dd_trace_internal_fn') && dd_trace_internal_fn('ddtrace_reload_config');
```

Comment:
Do we need this here since it's being called in **bridge/dd_wrap_autoloader.php**? I might be missing the context for this though. :)

Reply:
I assumed someone could require `dd_init` directly too - just to be safe I've added `ddtrace_reload_config` call here to make sure its not surprising for our users if their `putenv` config stopped working.